auth/auth_client.go

```diff
@@ -8,18 +8,14 @@ import (
 
 	"github.com/fabric8-services/fabric8-auth/errors"
 
+	commonConfig "github.com/fabric8-services/fabric8-common/configuration"
```

Comment:
ah, I forgot about this convention - thanks for pointing it out.
> Why not just "github.com/fabric8-services/fabric8-common/configuration" (== configuration)?

because I wanted to distinguish the tenant config and common config

I'll change it to `cfg`

Reply:
fixed in 1a94b7f